content/en/apps/guides/data/csv-to-docs.md

```diff
@@ -66,7 +66,7 @@ Converting that CSV file to JSON docs with the `csv-to-docs` action would genera
 }
 ```
 
-#### Special notations
+### Special notations
 
 ##### Specifying property type
```

Comment:
Updating the level here too:
```suggestion
#### Specifying property type
```

Reply:
My thought was that Special notations was the section and the others were sub categories underneath that. Do you think they should be retained as the way they are?

Reply:
I think we agree here... Special notations has 3 #, this subsection should have 4, not 5.

Reply:
oh! I was seeing 4 and not 5. I see the problem now.